zips/zip-0230.rst

```diff
@@ -480,6 +480,22 @@ An issuance note description, ``IssueNoteDescription`` contains the following fi
 |                             |                              |                                                |§3.2.1 ‘Note Plaintexts and Memo Fields’.                            |
 +-----------------------------+------------------------------+------------------------------------------------+---------------------------------------------------------------------+
 
+Orchard Note Plaintext
+----------------------
+
+Each Orchard note plaintext consists of
+
+.. math:: (\mathsf{leadByte} : \mathbb{B}^{\mathbb{Y}}, \mathsf{d} : \mathbb{B}^{[\ell_{\mathsf{d}}]}, \mathsf{v} : \{0 .. 2^{\ell_{\mathsf{value}}} - 1\}, \mathsf{rseed} : \mathbb{B}^{\mathbb{Y}[32]}, \mathsf{asset\_base} : \mathbb{B}^{[\ell_{\mathbb{P}}]}, \mathsf{K^{memo}} : \mathbb{B}^{\mathbb{Y}[32]})
+
+The encoding of an Orchard note plaintext [#protocol-noteptencoding]_ is updated and now consists of the following fields:
+
+$[\mathsf{leadByte}] \,||\, \mathsf{LEBS2OSP}_{88}(\mathsf{d}) \,||\, \mathsf{I2LEOSP}_{64}(\mathsf{v}) \,||\, \mathsf{rseed} \,||\, \mathsf{LEBS2OSP}_{256}(\mathsf{repr}_{\mathbb{P}}(\mathsf{AssetBase})) \,||\, \mathsf{K^{memo}}$
+
+* The $\mathsf{leadByte}$ MUST be encoded to be $\mathsf{0x03}$ for Orchard note plaintexts in V6 transactions. 
```

Comment:
```suggestion
* The $\mathsf{leadByte}$ MUST be $\mathtt{0x03}$ for Orchard note plaintexts in V6 transactions. 
```
It literally is $\mathsf{0x03}$, which is a byte value. To be pedantic, the encoding is the byte sequence $[\mathtt{0x03}]$, but there's no need to say that because line 492 says it.